docs/synapse/userguides/storm_ref_lift.rstorm

```diff
@@ -1,319 +1,402 @@
+.. storm-pre:: file:bytes:_virustotal:reputation
+::
+  
+  file:bytes:_virustotal:reputation
 
-.. storm-pre:: [inet:dns:a=(blackcake.net, 52.4.209.250)]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: inet:dns:a:fqdn = blackcake.net
+.. NOTE::
+  
+  The ``:_virustotal:reputation`` extended property is added to the Synapse data model by the `Synapse-VirusTotal`_ Power-Up.
 
 
-- Lift all the files with a PE compiled time of ``1992-06-19 22:22:17``:
+.. _lift-prop-standard:
 
+Lift by Property Value - Standard Comparison Operators
+------------------------------------------------------
 
-.. storm-pre:: [file:bytes=sha256:2d168c4020ba0136cd8808934c29bf72cbd85db52f5686ccf84218505ba5552e :mime:pe:compiled="1992/06/19 22:22:17.000"]
-.. storm-opts:: {"show": ["err", "fini", "init", "warn"]}
-.. storm-cli:: file:bytes:mime:pe:compiled = "1992/06/19 22:22:17"
+A "lift by property value" operation returns the node(s) whose property matches the specified value. This type of lift requires:
 
+- the form name or full property name (i.e., the combined form and property name) that you will use to lift the node(s);
+- a :ref:`gloss-comp-operator` to specify how the property value should be evaluated; and
+- the property value.
 
-- Lift all the files with a PE compiled time that falls within the year 2019:
+A "lift by property value" can be performed using primary, secondary, universal, or extended properties.
 
-.. storm-pre:: [ file:bytes=sha256:9f9d96e99cef99cbfe8d02899919a7f7220f2273bb36a084642f492dd3e473da :mime:pe:compiled='2019/01/01 11:14:00.000' ]
-.. storm-pre:: [ file:bytes=sha256:bd422f912affcf6d0830c13834251634c8b55b5a161c1084deae1f9b5d6830ce :mime:pe:compiled='2019/01/01 22:02:16.000' ]
+In Synapse, we define **standard comparison operators** as the following set of operators:
 
-.. storm-cli:: file:bytes:mime:pe:compiled=2019*
+- equal to ( ``=`` )
+- less than ( ``<`` )
+- greater than ( ``>`` )
+- less than or equal to ( ``<=`` )
+- greater than or equal to ( ``>=`` )
 
-- Lift all nodes created during July 2023:
+The :ref:`lift-try` ( ``?=`` ) can optionally be used in place of the standard equal to operator ( ``=`` ).
+Use of the try operator is generally not required for interactive Storm queries, but may be useful for more
+complex Storm queries (such as automation or Storm-based ingest queries).
 
-.. storm-cli:: .created=202307*
+The most commonly used standard comparison operator is the equal to ( ``=`` ) operator. Comparison operators that
+expect a **quantity** (i.e., the inequality symbools ``<``, ``>``, ``<=``, and ``>=``) can only be used with
+properties whose type supports the comparison (e.g., integers, dates/times, etc.)
 
-*Lift all nodes with a specific tag:*
+.. TIP::
+  
+  IP addresses (``inet:ipv4`` and ``inet:ipv6`` nodes) are stored as their decimal integer equivalents,
```

Comment:
IPv6 nodes are actually stored as strings currently due to their size (msgpack doesn't allow numbers that large).

Reply:
"The more you know..." Clarified. Thanks!